pyproject.toml

```diff
@@ -48,7 +48,8 @@ dependencies = [
     "pyts",
     "torch",
     "torchdiffeq",
-    "toml"
+    "toml", 
+    "lightning"
 ]
 
 version = "1.4.2"
```

Comment:
Do we need to update this manually or will it be bumped automatically on release?